libs/shared/infrastructure/di/src/lib/api-infrastructure-container.ts

```
@@ -0,0 +1,14 @@
+import { graphqlClient, UsersRepositoryImpl } from '@shared/infrastructure-api';
+import { UsersRepository } from '@users/domain';
+
+export class APIInfrastructureContainer {
+  private static client: typeof graphqlClient;
+  private static usersRepository: UsersRepository;
+
+  static getUsersRepository() {
+    if (!this.usersRepository) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)